pinot-controller/src/main/java/com/linkedin/pinot/controller/helix/core/realtime/PinotLLCRealtimeSegmentManager.java

```diff
@@ -1348,8 +1348,9 @@ private Exception getException() {
     @Override
     public Boolean call() throws Exception {
 
+      String clientId = PinotLLCRealtimeSegmentManager.class.getSimpleName() + "_" + _topicName + "_" + _partitionId;
```

Comment:
Is this an underscore as opposed to a dash in the PinotIdealStateBuilder as the separator?

Reply:
Fixed